src/helpers/passable_args/index.js

```diff
@@ -17,27 +17,51 @@
  * @param {Array.<{specific: String[], passes: passableCallback, custom: Object}>} args - arguments for Passable configuration
  * @return {object} Passable configuration settings
  */
-function passableArgs(args: PassableArguments) {
+function passableArgs(args: PassableArguments): PassableRuntime {
 
-    let passes: Function,
-        specific: specific = [],
+    let passes: Passes,
+        specific: Specific = [],
         custom: Rules = {};
 
-    switch (args.length) {
-      case 0:
-        throw new TypeError('[passable]: Failed to execute `passableArgs`: At least 1 argument required, but only 0 present.');
+    let res: [Array<string> | string, Passes, Rules];
 
-      case 1: // [passes] = args;
-        if (typeof args[0] != 'function') {
-          throw new TypeError('[passable]: Failed to execute `passableArgs`: Unexpected ' + typeof args[0] + ', expected function');
-        }
-        args = [specific, ...args];
+    switch (args.length) {
+        case 0:
+            throw new TypeError("[passable]: Failed to execute 'passableArgs': At least 1 argument required, but only 0 present.");
 
-      case 2:
-        args = (typeof args[1] == 'function') ? [...args, custom] : [specific, ...args];
+        case 1: // [passes] = args;
+            if (typeof args[0] !== 'function') {
+                throw new TypeError(`[passable]: Failed to execute 'passableArgs': Unexpected ${typeof args[0]}, expected function`);
```

Comment:
Maybe:
```
Expected Function, but got ${typeof args[0]}
```

Reply:
I believe the current phrasing fits more with the other errors thrown by Passable.